schemas/context/advertising.schema.json

```diff
@@ -31,18 +31,21 @@
           "$ref": "https://ns.adobe.com/xdm/context/advertising-timed-asset-view-details"
         },
         "xdm:completes": {
-          "title": "Completes",
-          "$ref": "https://ns.adobe.com/xdm/data/measure",
```

Comment:
@midomir This involves a breaking change. All the previous datasets become unusable with this change as we are trying to change the type of the property from "object" to "integer".